src/components/TransactionItem/TransactionIncome/styled.tsx

```diff
@@ -20,7 +20,7 @@ export const TransactionCapacityValuePanel = styled.div`
   display: flex;
   justify-content: flex-end;
   align-items: center;
-  color: ${(props: { increased: boolean; theme: any }) => (props.increased ? props.theme.primary : '#FF6347')};
+  color: ${(props: { increased: boolean; theme: any }) => (props.increased ? '#333' : '#FF6347')};
```

Comment:
I didn't see the change of color in the design draft.
But I do remember a blame that `capacity` in `primary color` looks like clickable. Is this change for that blame? If yes, please add a reference.